### PR TITLE
Update getting started guide to work correctly

### PR DESCRIPTION
**Pull request by torkleyy**
Fixes https://github.com/svgdotjs/svg.js/issues/1021

### CODE REVIEW

**Comment by larshp** [comment]
and it should be `'#drawing'` instead of `'drawing'` ?

File: content/3-getting-started/default.txt [hunk 1]
```diff
@@ -24,10 +24,10 @@ SVG.js assumes you have an HTML element with an `id` attribute created and ready
 
 ## Create an SVG document
 
-Next, use the `SVG()` function to create an SVG document within the wrapper element:
+Next, use the `SVG()` function and its `addTo` method to create an SVG document within the wrapper element:
 
 ```javascript
-var draw = SVG('drawing').size(300, 300)
+var draw = SVG().addTo('drawing').size(300, 300)
```
Comment (torkleyy) [inline content/3-getting-started/default.txt]:
```suggestion
var draw = SVG().addTo('#drawing').size(300, 300)
```

File: content/3-getting-started/default.txt [hunk 2]
```diff
@@ -43,7 +43,7 @@ The first argument can either be an id of the element or the selected element it
 By default the svg drawing follows the dimensions of its parent, in this case `#drawing`:
 
 ```javascript
-var draw = SVG('drawing').size('100%', '100%')
+var draw = SVG().addTo('drawing').size('100%', '100%')
```
Comment (torkleyy) [inline content/3-getting-started/default.txt]:
```suggestion
var draw = SVG().addTo('#drawing').size('100%', '100%')
```

File: content/3-getting-started/default.txt [hunk 3]
```diff
@@ -52,7 +52,7 @@ By default, SVG.js assumes the client's browser supports SVG. You can test suppo
 
 ```javascript
 if (SVG.supported) {
-  var draw = SVG('drawing')
+  var draw = SVG().addTo('drawing')
```
Comment (torkleyy) [inline content/3-getting-started/default.txt]:
```suggestion
  var draw = SVG().addTo('#drawing')
```

File: content/3-getting-started/default.txt [hunk 4]
```diff
@@ -66,7 +66,7 @@ This might seem obvious to many but it's still worth mentioning. If you include
 
 ```javascript
 SVG.on(document, 'DOMContentLoaded', function() {
-  var draw = SVG('drawing')
+  var draw = SVG().addTo('drawing')
```
Comment (torkleyy) [inline content/3-getting-started/default.txt]:
```suggestion
  var draw = SVG().addTo('#drawing')
```

File: content/3-getting-started/default.txt [hunk 5]
```diff
@@ -82,7 +82,7 @@ SVG.js also works outside of the HTML DOM, inside an SVG document for example:
   <script type="text/javascript" xlink:href="svg.min.js"></script>
   <script type="text/javascript">
     <![CDATA[
-      var draw = SVG('drawing')
+      var draw = SVG().addTo('drawing')
```
Comment (torkleyy) [inline content/3-getting-started/default.txt]:
```suggestion
      var draw = SVG().addTo('#drawing')
```

Comment (Fuzzyma) [reply content/3-getting-started/default.txt]: This one is not correct. When you use svg.js in an SVG document you dont want to create a new SVG in the existing one. You mostly only want to get the reference of the SVG itself. 
Thats why you would use the syntax: `SVG('#drawing')` which gives you back the svg.js object for the root svg

File: content/3-getting-started/default.txt [hunk 6]
```diff
@@ -82,7 +82,7 @@ SVG.js also works outside of the HTML DOM, inside an SVG document for example:
   <script type="text/javascript" xlink:href="svg.min.js"></script>
   <script type="text/javascript">
     <![CDATA[
-      var draw = SVG('drawing')
+      var draw = SVG().addTo('#drawing')
```
Comment (torkleyy) [inline content/3-getting-started/default.txt]:
```suggestion
      var draw = SVG('#drawing')
```

**Comment by qichunren** [comment]
Please update documentation in website, it takes me 1 hour to debug.

File: content/3-getting-started/default.txt (hunk 5 above)
Comment (jobsFu) [reply content/3-getting-started/default.txt]: It's so unreliable you have to learn to appreciate the fact that you just fucking follow the document and report an error

**Comment by Fuzzyma** [comment]
New docs available now

**Pull request closed without merging**